templates/csharp/command-and-response/teamsapp.local.yml.tpl

```diff
@@ -15,30 +15,24 @@ provision:
       teamsAppId: TEAMS_APP_ID
 
   # Create or reuse an existing Microsoft Entra application for bot.
-  - uses: botAadApp/create
+  - uses: aadApp/create
     with:
       # The Microsoft Entra application's display name
       name: {{appName}}${{APP_NAME_SUFFIX}}
+      generateClientSecret: true
+      signInAudience: AzureADMultipleOrgs
     writeToEnvironmentFile:
       # The Microsoft Entra application's client id created for bot.
-      botId: BOT_ID
+      clientId: BOT_ID
       # The Microsoft Entra application's client secret created for bot.
-      botPassword: SECRET_BOT_PASSWORD
+      clientSecret: SECRET_BOT_PASSWORD
+      # The Microsoft Entra application's object id created for bot.
+      objectId: BOT_OBJECT_ID
 
   # Generate runtime appsettings to JSON file
   - uses: file/createOrUpdateJsonFile
     with:
-{{#isNewProjectTypeEnabled}}
```

Comment:
Mis-removed? Or is this not in the template mustache?

Reply:
It's inconsistent with the template mustache and my local changes that revert these change were not pushed along with this PR. Updated. Thank you.